### PR TITLE
Make tab shortcut close showWpm when finish

### PR DESCRIPTION
**Pull request by ronnapatp**
Make tab button close show wpm modal and reset word

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -159,6 +159,12 @@
         showMenu = true
       }
     }
+    if (e.key === 'Tab') {
+      if (showWpm = true){
```
Comment (tinvv) [inline src/routes/index.svelte:163]: should it be === instead = ?

Comment (ronnapatp) [reply src/routes/index.svelte]: OK

Comment (ronnapatp) [reply src/routes/index.svelte]: OK

File: src/routes/index.svelte [hunk 2]
```diff
@@ -159,6 +159,12 @@
         showMenu = true
       }
     }
+    if (e.key === 'Tab') {
+      if (showWpm === true){
+        showWpm === false
```
Comment (narze) [inline src/routes/index.svelte:164]:
```suggestion
        showWpm = false
```

**Review by narze — CHANGES_REQUESTED** [review]
^

File: src/routes/index.svelte (hunk 2 above)
Comment (narze) [reply src/routes/index.svelte:164]: What does this code do I'm not sure, try to explain it in PR description

**Comment by ronnapatp** [comment]
Done

**Review by narze — APPROVED** [review]

**Pull request merged**